packages/hardhat-core/src/internal/cli/project-creation.ts

```diff
@@ -245,6 +245,14 @@ async function createPackageJson() {
   );
 }
 
+function showStarOnGitHubMessage() {
+  console.log(
+    chalk.cyan("Give Hardhat a star on Github if you're enjoying it") +
+      emoji(" 💞✨")
+  );
```

Comment:
```suggestion
  console.log(
    chalk.cyan("Give Hardhat a star on Github if you're enjoying it!") +
      emoji(" 💞✨")
  );
  console.log();
```

Reply:
The exclamation mark makes it look a bit better in terminals that don't support emoji.